Template/Trigger/UserInteractionTrigger.web.js

```diff
@@ -0,0 +1,30 @@
+(function () {
+    return function (parameters, TagManager) {
+        this.setUp = function (triggerEvent) {
+
+            const eventNames = [
+                "touchstart",
+                "mouseover",
+                "wheel",
+                "scroll",
+                "keydown"
+            ];
+            
+            const init = () => {
+                triggerEvent({event: 'UserInteraction'});
+                removeListeners();
+            }
+            
+            const removeListeners = () => {
+                for (var i = 0, iLen = eventNames.length; i < iLen; i++) {
+                    window.removeEventListener(eventNames[i], init);
```

Comment:
We do not use `window` directly and use `windowAlias` instead, which you can define at the start using below code
```
var windowAlias = parameters.window;
```